components/dashboard/src/repositories/list/RepoListEmptyState.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FC } from "react";
+import { Heading2, Subheading } from "@podkit/typography/Headings";
+import { Button } from "@podkit/buttons/Button";
+import { cn } from "@podkit/lib/cn";
+
+type Props = {
+    onImport: () => void;
+};
+export const RepoListEmptyState: FC<Props> = ({ onImport }) => {
+    return (
+        <div className={cn("w-full flex justify-center mt-2 rounded-xl bg-gray-100 dark:bg-gray-800 px-4 py-20")}>
+            <div className="flex flex-col justify-center items-center text-center space-y-4">
+                <Heading2>No repositories yet</Heading2>
+                <Subheading className="max-w-md">
```

Comment:
Could we change the color in dark mode for more contrast?
<img width="480" alt="image" src="https://github.com/gitpod-io/gitpod/assets/29888641/24c3ff62-9667-4735-84d0-5bfaf2cf7e48">

Reply:
It applies to this as well 🙏 

<img width="514" alt="image" src="https://github.com/gitpod-io/gitpod/assets/
29888641/28ffafc1-80fb-4041-a809-bb59a3e7ca43">

Reply:
Good catch. I think I'm going to look at swapping all our headings to the podkit ones to at least get that stuff cleaned up and using the updated components.